meetings/2017-05-22.md

```diff
@@ -0,0 +1,91 @@
+# Node.js Foundation Community Committee Meeting 2017-05-22
+
+## Links
+
+* **GitHub Issue**: [community-committee#62](https://github.com/nodejs/community-committee/issues/62)
+* **Minutes Google Doc**: https://docs.google.com/document/d/1urhinDNJK-EEMziNob8-mlsOEZM_zecluDIgBA6kVOw/
+* _Previous Minutes Google Doc_: https://docs.google.com/document/d/1YevHuA_csoTltDJ6mHr21fetZshNqWpsRbBRkSHhmYE/
+
+## Present
+
+* Bryan Hughes @nebrius (CommComm)
+* Gregor Martynus @gr2m (CommComm)
+* Jenn Turner @renrutnnej (observer)
+* Penny Hugger @pup (observer)
+* Rachel White @rachelnicole (CommComm)
+* Tierney Cyren @bnb (observer)
+* Tracy Hinds @hackygolucky (CommComm)
+* William Kapke @williamkapke (CommComm)
+* Michael Dawson (observer, TSC)
+* Sarah Conway (observer, LF)
+* Emily Rose @nexxy (CommComm)
+
+## Agenda
+
+Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/TSC
+
+* Request to move moderation responsibilities to CommComm [#270](https://github.com/nodejs/TSC/issues/270)
+
+### nodejs/community-committee
+
+* Charter tweaks [#50](https://github.com/nodejs/community-committee/pull/50)
+* Expanding the Node.js Collection: Reviewers [#45](https://github.com/nodejs/community-committee/issues/45)
+* Adding folks as members [#38](https://github.com/nodejs/community-committee/issues/38)
+* Moderation shortage [#33](https://github.com/nodejs/community-committee/issues/33)
+
+## Review of last meeting
+
+Skipped because we started the meeting late.
+
+## Minutes
+
+### nodejs/TSC
+
+#### Request to move moderation responsibilities to CommComm [#270](https://github.com/nodejs/TSC/issues/270)
+
+This is the formal request to move moderation responsibilities over to the Community Committee. Concerns were brought up that nothing is changing so why is it moving? Skills and scope of these different committees makes sense to expand the coverage of moderation to all repos(not just TSC overseen repos but also CommComm and even things such as IRC potentially). Will allow TSC to not have to spend energy on this as much but everyone is covered under CommComm, including TSC, to the voices are represented. No objections.
+
+### nodejs/community-committee
+
+#### Charter tweaks [#50](https://github.com/nodejs/community-committee/pull/50)

```

Comment:
unless it had any impact on what came after it (which i don't think it did?) i think we can leave this be